index.html

```diff
@@ -138,11 +138,15 @@
                     <div class="title" data-i18n="profile.title"></div>
                     <div class="sideToolbarBlock first">
                         <label class="first" data-i18n="profile.setDisplayNameLabel"></label>
-                        <input type="text" id="setDisplayName" data-i18n="[placeholder]settings.name" placeholder="Name">
+                        <div class="input-control__container">
+                            <input class="input-control__input" type="text" id="setDisplayName" data-i18n="[placeholder]settings.name" placeholder="Name">
```

Comment:
I don't understand why we would need to wrap every input in a container. All inputs have the same css class anyway, why don't we specify the font size there? Maybe there's something I'm missing, but this doesn't seem the best way to address the issue.